org.jacoco.core/src/org/jacoco/core/runtime/AgentModule.java

```diff
@@ -0,0 +1,151 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2023 Mountainminds GmbH & Co. KG and Contributors
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.runtime;
```

Comment:
Should be in package `org.jacoco.agent.rt.internal`

Reply:
Done. But now we have a dependency on internal core class `org.jacoco.core.internal.InputStreams`. Technically this is no problem as the agent is shaded anyways. Or should I better duplicate the code and avoid the dependency?

Reply:
IMO ok as it is.